testing/tests/DevExpress.ui.widgets/pager.tests.js

```diff
@@ -1350,8 +1472,8 @@ QUnit.module('Pager', {
         $pager.width(pager._$pagesSizeChooser.width() + pager._$pagesChooser.width() + infoWidth + 50);
         pager._dimensionChanged();
 
-        assert.ok(!pager.option('lightModeEnabled'), 'lightModeEnabled');
-        assert.notStrictEqual(pager._$info.css('display'), 'none', 'info element is shown');
+        assert.ok(!isLightMode(pager), 'lightModeEnabled');
+        assert.ok(pager._$info.length === 1 || pager._$info.css('display') !== 'none', 'info element is hidden');
```

Comment:
|| or &&? hidden or visible?

Reply:
newpager  = pager._$info.length === 1) 
oldpager == pager._$info.css('display') !== 'none') == assert.notStrictEqual(pager._$info.css('display'), 'none', 'info element is shown')
So oldPager || newPager